### PR TITLE
Fix unreliable WebConsoleTests

### PR DESCRIPTION
**Pull request by k-wall**

### CODE REVIEW

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

File: systemtests/src/main/java/io/enmasse/systemtest/platform/OpenShift.java [hunk 1]
```diff
@@ -42,7 +43,14 @@ public OpenShift(Environment environment, String globalNamespace) {
...
             // Workaround https://github.com/square/okhttp/issues/3146
-            httpClient = httpClient.newBuilder().protocols(Collections.singletonList(Protocol.HTTP_1_1)).build();
+            httpClient = httpClient.newBuilder()
+                    .protocols(Collections.singletonList(Protocol.HTTP_1_1))
+                    .connectTimeout(60, TimeUnit.SECONDS)
+                    .writeTimeout(60, TimeUnit.SECONDS)
+                    .readTimeout(60, TimeUnit.SECONDS)
+                    .build();
+
+            log.info("KWDEBUG retryOnConnectionFailure", httpClient.retryOnConnectionFailure());
```
Comment (famartinrh) [inline systemtests/src/main/java/io/enmasse/systemtest/platform/OpenShift.java]: don't forget to delete this line :)

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

File: systemtests/src/main/java/io/enmasse/systemtest/platform/OpenShift.java (hunk 1 above)
Comment (k-wall) [reply systemtests/src/main/java/io/enmasse/systemtest/platform/OpenShift.java]: Done

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxWebConsoleTest

**Pull request merged**